package.json

```
@@ -9,7 +9,7 @@
     "server",
     "email"
   ],
-  "version": "2.8.27",
+  "version": "2.8.28",
```

```suggestion
  "version": "2.8.27",
```

this PR won't likely drive a Haraka release.